.github/ISSUE_TEMPLATE/first_issue.md

```diff
@@ -33,7 +33,7 @@ You do not need to know anything prior to taking up this issue! Welcome to opens
 
 If you are not familiar with GitHub or how to create a pull request, [here is a guide you can follow on how GitHub works.](https://guides.github.com/activities/hello-world/)
 
-If you are familiar with the terminal or would like to learn to use it, [here is a great tutorial on how to send a pull request using the terminal.](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github)
+If you are familiar with the terminal or would like to learn to use it, [here is a great tutorial on how to send a pull request using the terminal.](https://www.youtube.com/watch?v=rgbCcBNZcdQ&t=8s)
```

Comment:
Hey, Sometimes it is easier to view a doc than a video, I suggest that you keep both the new doc link (which is in the issue) as well as the video so it can suit the contributor's need.

Reply:
I was thinking the same. Can you add the video instead replacing the old link?

Reply:
https://guides.github.com/activities/forking/ may be this one?

Reply:
> https://guides.github.com/activities/forking/ may be this one?

Hey @orensbruli sir, thank you so much for your advice. I'll update it and I'll let you know.

Reply:
> Hey, Sometimes it is easier to view a doc than a video, I suggest that you keep both the new doc link (which is in the issue) as well as the video so it can suit the contributor's need.

Hey, @pawanw17 I'll add the new doc link as you mentioned.
Thank you so much for your advice.

Reply:
@orensbruli this is the link https://github.com/robocomp/robocomp/blob/development/doc/contribute/contribute.md